Tests/Cell.py

```diff
@@ -12,11 +12,29 @@
 from TM1py.Services import TM1Service
 from TM1py.Utils import Utils, element_names_from_element_unique_names
 
+try:
+    import pandas as pd
+
+    _has_pandas = True
+except ImportError:
+    _has_pandas = False
+
+def skipIfNoPandas(func):
```

Comment:
Good idea to use a decorator here.
Please change the name to `skip_if_no_pandas` for the sake of consistency.